scripts/quickstart.sh

```diff
@@ -139,27 +138,65 @@ start() {
     compose_cmd=$(get_compose_cmd)
 
     # Launch OPA
-    echo "Starting Superblocks On-Premise-Agent..."
-    curl -s "$1" | ${compose_cmd} -p superblocks --env-file "$2" -f - up > "$3" &
-}
+    if [ $4 = 1 ]; then
```

Comment:
Even though the variable `$4` is a boolean, I think it would be better to handle it defensively, i.e.
```
if [[ $4 = 1 ]]; then
```
or
```
if [ "$4" = 1 ]; then
```